### PR TITLE
decompile pokeblock files

### PR DESCRIPTION
**Pull request by DizzyEggg**
Left out use pokeblock.s.

### CODE REVIEW

File: src/pokeblock.c [hunk 1]
```diff
@@ -0,0 +1,1337 @@
...
+}
+
+static void sub_8136344(void)
+{
+    if (sSavedPokeblockData.lastItemPos > 4)
+    {
+        u8 i;
+
+        for (i = 0;
+             i < sSavedPokeblockData.lastItemPos - 4 && sSavedPokeblockData.lastItemPage + sPokeblockMenu->maxShowed != sPokeblockMenu->itemsNo;
```
Comment (Diegoisawesome) [inline src/pokeblock.c]: Parens around different parts of statement

File: src/pokeblock_feed.c [hunk 2]
```diff
@@ -0,0 +1,1108 @@
...
+    gTasks[taskId].func = Task_WaitForAtePokeblockText;
+}
+
+static void Task_ReturnAfterPaletteFade(u8 taskId)
+{
+    if (!gPaletteFade.active)
+    {
+        ResetSpriteData();
+        FreeAllSpritePalettes();
+        m4aMPlayVolumeControl(&gMPlay_BGM, -1, 256);
```
Comment (Diegoisawesome) [inline src/pokeblock_feed.c]: 0x100

File: src/pokeblock_feed.c [hunk 3]
```diff
@@ -0,0 +1,1108 @@
...
+
+        if (!var_24)
+        {
+            pokeblockFeed->field_850[r4] = Sin(pokeblockFeed->field_1060[0], pokeblockFeed->field_1060[2] + r5 / 256) + r8;
+            pokeblockFeed->field_C50[r4] = Cos(pokeblockFeed->field_1060[0], pokeblockFeed->field_1060[3] + r5 / 256) + r7;
+        }
+        else
+        {
+            pokeblockFeed->field_850[r4] = Sin(pokeblockFeed->field_1060[0], pokeblockFeed->field_1060[2] - r5 / 256) + r8;
+            pokeblockFeed->field_C50[r4] = Cos(pokeblockFeed->field_1060[0], pokeblockFeed->field_1060[3] - r5 / 256) + r7;
```
Comment (Diegoisawesome) [inline src/pokeblock_feed.c]: All these 256s to 0x100

File: src/use_pokeblock.c [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+EWRAM_DATA struct Pokeblock *gUnknown_0203BC98 = NULL;
+EWRAM_DATA u8 gPokeblockMonId = 0;
+EWRAM_DATA s16 gPokeblockGain = 0;
+EWRAM_DATA void *gUnknown_0203BCA0 = NULL;
+EWRAM_DATA void *gUnknown_0203BCA4 = NULL;
+EWRAM_DATA void *gUnknown_0203BCA8 = NULL;
+EWRAM_DATA struct UsePokeblockStruct *gUnknown_0203BCAC = NULL;
+
+// const rom data
+// todo: make it static once the file is decompiled
```
[the remark this answers is not in the document]
Comment (DizzyEggg) [reply src/use_pokeblock.c]: I'm leaving the 'use pokeblock' file.

[the remark this answers is not in the document]
Comment (Diegoisawesome) [reply src/use_pokeblock.c]: I don't understand what that means.

**Pull request merged**